[2024-01-17T21:03:14.447-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-18T00:01:59.135588+00:00 [queued]>
[2024-01-17T21:03:14.450-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-18T00:01:59.135588+00:00 [queued]>
[2024-01-17T21:03:14.450-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T21:03:14.470-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-18 00:01:59.135588+00:00
[2024-01-17T21:03:14.474-0300] {standard_task_runner.py:57} INFO - Started process 106107 to run task
[2024-01-17T21:03:14.476-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_coleta_teste', 'df', 'scheduled__2024-01-18T00:01:59.135588+00:00', '--job-id', '2909', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpela7crx7']
[2024-01-17T21:03:14.476-0300] {standard_task_runner.py:85} INFO - Job 2909: Subtask df
[2024-01-17T21:03:14.510-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_coleta_teste.df scheduled__2024-01-18T00:01:59.135588+00:00 [running]> on host desktop
[2024-01-17T21:03:14.575-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_coleta_teste' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-18T00:01:59.135588+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-18T00:01:59.135588+00:00'
[2024-01-17T21:03:18.018-0300] {logging_mixin.py:154} INFO - Called | 1705536198.015927 | df
[2024-01-17T21:03:18.026-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-17T21:03:18.027-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T21:03:18.029-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_coleta_teste, task_id=df, execution_date=20240118T000159, start_date=20240118T000314, end_date=20240118T000318
[2024-01-17T21:03:18.054-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T21:03:18.062-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
